DEVELOPING_OPAL/demos/src/main/scala/org/opalj/ai/domain/l1/IfNullParameterAnalysis.scala

```diff
@@ -45,107 +46,99 @@ import scala.collection.parallel.CollectionConverters.ImmutableIterableIsParalle
  */
 object IfNullParameterAnalysis extends ProjectAnalysisApplication {
 
-    override def title: String =
-        "Identifies methods that are sensitive to parameters that are \"null\""
+    override def title: String = "Identifies methods that are sensitive to parameters that are \"null\""
 
-    override def description: String =
-        "Identifies methods that exhibit different behavior w.r.t. "+
-            "the number and kind of thrown exceptions if a parameter is \"null\"."
+    override def description: String = "Identifies methods that exhibit different behavior w.r.t. " +
+        "the number and kind of thrown exceptions if a parameter is \"null\"."
 
     override def doAnalyze(
         theProject:    Project[URL],
         parameters:    Seq[String],
-        isInterrupted: () => Boolean
-    ): BasicReport = {
+        isInterrupted: () => Boolean): BasicReport = {
 
         // Explicitly specifies that all reference values are not null.
-        def setToNonNull(domain: DefaultDomain[URL])(locals: domain.Locals): domain.Locals = {
-            locals.map { value =>
-                if (value == null)
-                    // not all local values are used right from the beginning
-                    null
-                else if (!value.isInstanceOf[domain.SingleOriginReferenceValue]) {
-                    // we are not concerned about primitive values
-                    value
-                } else {
-                    val initialReferenceValue = value.asInstanceOf[domain.SingleOriginReferenceValue]
-                    // create a new reference value which is not null
-                    initialReferenceValue.update(isNull = No)
-                }
+        def setToNonNull(domain: DefaultDomain[URL])(locals: domain.Locals): domain.Locals = locals.map { value =>
```

Comment:
Not very nice. Can we avoid this?

Reply:
I don't know. Maybe if we change the redundant brace rules?

Reply:
What in particular is here not very nice for you?

Reply:
It forced the map out of braces, making this a single-statement method. Though upon a second look, it's probably fine.